src/cmd/builtin/config.c

```diff
@@ -192,6 +195,60 @@ static int internal_config_reload (optparse_t *p, int ac, char *av[])
     return (0);
 }
 
+static int config_load (optparse_t *p, int ac, char *av[])
+{
+    int index = optparse_option_index (p);
+    const char *path = NULL;
+    ssize_t n;
+    void *buf;
+    json_t *obj;
+    flux_t *h;
+    flux_future_t *f;
+
+    if (index < ac)
+        path = av[index++];
+    if (index != ac) {
+        optparse_print_usage (p);
+        exit (1);
+    }
+    if (path) {
+        flux_conf_t *conf;
+        flux_error_t error;
+
+        if (!(conf = flux_conf_parse (path, &error))
+            || flux_conf_unpack (conf, &error, "O", &obj) < 0)
+            log_msg_exit ("Error parsing config: %s", error.text);
+        flux_conf_decref (conf);
+    }
+    else {
+        if ((n = read_all (STDIN_FILENO, &buf)) < 0)
```

Comment:
can move `buf` and `size` vars into this block b/c its only used here

Reply:
Good catch.  OK, setting MWP - thanks!